### PR TITLE
Add subnet ids and security group ids to dev-private & prod-private

### PR DESCRIPTION
**Pull request by TiannaW24**
### JIRA Ticket

https://securedatacommons.atlassian.net/browse/SDC-2579

### Summary

Attach our dev-private and production-private webportal lambdas to the VDMS VPC

### CODE REVIEW

File: webportal/lambda/.chalice/config.json [hunk 1]
```diff
@@ -50,7 +50,9 @@
         "TABLENAME_MANAGE_UPTIME_INDEX":"dev-scheduleuptime-username-index",
         "COGNITO_USER_POOL":"dev-sdc-dot-cognito-pool",
         "IDP_PROVIDER_ARNS":"arn:aws:cognito-idp:us-east-1:911061262852:userpool/us-east-1_RqSIpOZN3"
-      }
+      },
+      "subnet_ids": ["subnet-1d181f40", "subnet-b29f87d6"],
+      "security_group_ids": ["sg-98eae8ee"]
     },
```
Comment (TiannaW24) [inline webportal/lambda/.chalice/config.json:56]: Holding off on prod until the CRB is approved

Comment (JAMSUPREME) [reply webportal/lambda/.chalice/config.json:56]: Where/how are these used? Chalice uses them?

Comment (TiannaW24) [reply webportal/lambda/.chalice/config.json:56]: https://aws.github.io/chalice/topics/configfile.html#subnet-ids

Comment (aaronFranssell) [reply webportal/lambda/.chalice/config.json:56]: If you are just going to wait for the CRB, can you make the prod changes too?

**Pull request merged**